### PR TITLE
Some easy clean up from previous secp256k1 changes

### PR DESCRIPTION
**Pull request by 0xjinn**
### Description

Some easy clean up from PRs, where it was merged before I update some fixes from comment - https://github.com/aptos-labs/aptos-core/pull/10410

### CODE REVIEW

**Comment by 0xjinn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10414** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-core/10414" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/aptos-labs/aptos-core/10414?utm_source=stack-comment).

**Review by 0xmaayan — APPROVED** [review]

**Review by heliuchuan — APPROVED** [review]

File: ecosystem/typescript/sdk_v2/src/core/account.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ export class Account {
    * Derives an account with random private key and address
    *
    * @param args.scheme SigningScheme - type of SigningScheme to use.
-   * Currently only Ed25519, MultiEd25519, and Secp256k1 are supported
+   * Currently only Ed25519, and Secp256k1 are supported
```
Comment (xbtmatt) [inline ecosystem/typescript/sdk_v2/src/core/account.ts:80]:
```suggestion
   * Currently Ed25519, and Secp256k1 are supported
```

nit but remove `only` so people don't think the SDK is incomplete or something

Comment (xbtmatt) [reply ecosystem/typescript/sdk_v2/src/core/account.ts:80]: Wait why did you remove `MultiEd25519` here? It should be here I think

File: ecosystem/typescript/sdk_v2/src/types/index.ts [hunk 2]
```diff
@@ -845,9 +845,9 @@ export type TableItemRequest = {
 export type AuthenticationKeyScheme = SigningScheme | DeriveScheme;
 
 /**
- * A list of signing schemes that are supported by Aptos.
+ * A list of signing schemes that are supported by Aptos. Reference from Rust
```
Comment (xbtmatt) [inline ecosystem/typescript/sdk_v2/src/types/index.ts:848]:
```suggestion
 * A list of signing schemes that are supported by Aptos. It serves as a domain separator
 * when hashing a public key to derive its corresponding authentication key.
 ```

**Review by xbtmatt — APPROVED** [review]

**Comment by 0xjinn** [comment]
Repo moved. CLosing this right now

**Pull request closed without merging**